spec/controllers/admin_posts_controller_spec.rb

```diff
@@ -27,4 +27,56 @@
       end
     end
   end
+
+  describe 'POST #update' do
```

Comment:
Could you switch the single quotes to double quotes so we're consistent throughout the file?